Readme.md

```diff
@@ -225,6 +225,14 @@ disk-opts:
     - sda3
 ```
 
+Disk-opts, names field support now regular expression to white-list disks names based on [golang MatchString](https://golang.org/pkg/regexp/#MatchString) implementation. However to use a regular expression you need to prefix the string value by a `~`. To whitelist all eth interfaces, you can set:
```

Comment:
You can use a more standard version of regex with https://golang.org/pkg/regexp/#CompilePOSIX, with it, it won't be related with Go impl

Reply:
This RegExp is already used on net iterfaces